pkg/flow/tracer_map.go

```diff
@@ -118,20 +118,6 @@ func (m *MapTracer) evictFlows(ctx context.Context, forwardFlows chan<- []*Recor
 	mtlog.Debugf("%d flows evicted", len(forwardingFlows))
 }
 
-func (m *MapTracer) aggregate(metrics []ebpf.BpfFlowMetrics) ebpf.BpfFlowMetrics {
-	if len(metrics) == 0 {
-		mtlog.Warn("invoked aggregate with no values")
-		return ebpf.BpfFlowMetrics{}
-	}
-	aggr := ebpf.BpfFlowMetrics{}
-	for _, mt := range metrics {
-		// eBPF hashmap values are not zeroed when the entry is removed. That causes that we
-		// might receive entries from previous collect-eviction timeslots.
-		// We need to check the flow time and discard old flows.
-		if mt.StartMonoTimeTs <= m.lastEvictionNs || mt.EndMonoTimeTs <= m.lastEvictionNs {
-			continue
-		}
-		Accumulate(&aggr, &mt)
-	}
-	return aggr
+func (m *MapTracer) aggregate(metric ebpf.BpfFlowMetrics) ebpf.BpfFlowMetrics {
```

Comment:
why is it necessary to keep this function?

Reply:
not needed let me remove it and remove reference to it in ut